packages/experiments/src/components/PersonaCoin/CoinAlternativeForSmallestSize/CoinAlternativeForSmallestSize.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Icon } from 'office-ui-fabric-react';
+
+export default class PersonaCoinSize10 extends React.Component {
+  private styles = { root: { fontSize: 10 } };
+
+  public shouldComponentUpdate(): boolean {
+    return false;
+  }
```

Comment:
If I understand correctly this is here because the props will never change. I think a better alternative may be to just make this a `PureComponent` instead of implementing `shouldComponentUpdate`, which should achieve the same result.

Reply:
Or maybe better yet, just make this a stateless component. There is some overhead to having this be a class, as well. I think this is one of those cases where a stateless component would be best.

Reply:
A stateless component wouldn't prevent that the whole subtree would need to re-render. I guess it's ok since this one is just the Icon. Will change :)